plugins/modules/snap.py

```diff
@@ -29,10 +30,12 @@
     name:
         description:
             - Name of the snaps to be installed.
-            - Any named snap accepted by the C(snap) command is valid.
+            - Any named snap accepted by the C(snap install) command is valid.
+              You can verify if a given snap is available on the Snap Store in advance by running: C(snap info <snap-name>).
             - >
-              Notice that snap files might require O(dangerous=true) to ignore the error
-              "cannot find signatures with metadata for snap".
+              When installing locally built `.snap` SquashFS files or downloading them from
```

Comment:
```suggestion
              When installing locally built C(.snap) SquashFS files or downloading them from
```
Check out https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation and the sections below for more formatting info.

Reply:
Fixed! Thank you!